lib/gui/os/window-progress.js

```diff
@@ -46,7 +47,8 @@ exports.currentWindow = electron.remote.getCurrentWindow();
 exports.set = (percentage, status) => {
   exports.currentWindow.setProgressBar(utils.percentageToFloat(percentage));
   const statusString = status === 'check' ? 'Validating' : 'Flashing';
-  exports.currentWindow.setTitle(`Etcher – ${statusString} ${percentage}%`);
+  const dash = '\u2013';
```

Comment:
BTW, should this variable be in uppercase (e.g. `DASH`), given that its statically determined? Also, can we call it something like `DASH_UNICODE_CHARACTER`, `SEPARATOR_UNICODE_CHARACTER`, or something like that?